woocommerce/woocommerce-admin
# Add WooCommerce flyout menu

Fixes #6129 

* Adds some helper methods to the menu
* Adds the top-level items from the main menu (with some exceptions from the design)

### Screenshots
<img width="348" alt="Screen Shot 2021-01-22 at 6 09 17 PM" src="https://user-images.githubusercontent.com/10561050/105558729-3adcaa80-5cdd-11eb-89c0-41dbe5f0aa7e.png">


### Detailed test instructions:

1. Enable the new navigation.
2. Hover the "WooCommerce" menu item.
3. Check that expected items appear.
4. Check that clicking those items goes to the correct page.
